config/devices.yml

```diff
@@ -1,41 +1,40 @@
 solar_panels:  
   config:    
     lander_pitch: 0.0
-    lander_roll: 0.0
-    lander_yaw: 0.0    
+    lander_roll:  0.0
+    lander_yaw:   0.0    
     surface_temperature_table: reference/lunarTemperature30degDiviner.txt
     PV_efficiency:
       temp: -173.15 20.0  126.85
       power: 0.356414 0.304828 0.257931
-    
-    
-  EPanel:
-    normal: 1 0 0 
-    efficiency: 1.0
-    area: 0.1565
   
-  WPanel:
-    normal: -1 0 0 
-    efficiency: 1.0
-    area: 0.1565
+  panels:
+    - name:       EPanel
+      normal:     1 0 0 
+      efficiency: 1.0
+      area:       0.1565
   
-  TPanel:
-    normal: 0 0 1
-    efficiency: 1.0
-    area: .313
-
+    - name:       WPanel
```

Comment:
To make this thing more consistent with the rest (i.e. `power_consumers` etc),  can we change the syntax to be
```
solar_panels:
    panels:
       EPanel:
            normal: 1 0 0 
```
etc? 
(rather than `- name: EPanel`)

Reply:
Done, pushed to branch, working on other items now.